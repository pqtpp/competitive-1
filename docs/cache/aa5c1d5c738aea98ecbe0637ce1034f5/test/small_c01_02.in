88 97
7 37 1
7 85 0
37 79 0
85 81 0
37 5 1
81 40 1
40 86 0
81 82 1
40 10 0
79 4 1
37 47 1
4 38 0
86 3 1
82 75 1
86 28 0
7 49 0
10 62 1
79 25 1
38 1 0
85 35 0
47 65 0
49 76 1
35 54 0
25 63 0
54 53 1
81 58 0
38 72 0
10 68 0
47 34 0
4 8 1
47 0 0
3 83 0
34 74 1
3 52 0
74 71 0
58 56 1
83 67 0
28 19 0
8 36 1
76 31 1
34 46 0
47 69 0
71 29 1
85 12 1
53 24 1
1 55 0
65 22 1
72 59 1
19 77 1
19 41 1
25 13 0
59 14 0
85 16 0
86 39 0
67 87 0
67 61 1
54 57 1
77 11 1
35 15 0
53 60 1
3 48 1
12 80 1
14 6 0
29 30 0
19 42 0
1 17 0
17 2 0
10 20 0
71 32 1
2 78 0
48 21 1
85 51 1
80 73 1
76 66 1
10 23 0
78 84 0
10 27 1
59 18 0
59 26 1
32 50 0
50 64 0
42 44 1
56 9 1
78 45 1
85 33 1
4 70 1
15 43 0
8 33 0
64 61 0
2 0 0
75 27 0
11 23 1
61 28 0
23 81 0
81 65 1
85 78 1
18 79 0
