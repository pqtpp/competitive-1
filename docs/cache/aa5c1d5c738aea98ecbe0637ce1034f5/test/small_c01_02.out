35
30 20 28 40 41 89 66 65 18 11 26 45 69 75 9 2 96 77 51 62 0 1 3 15 19 22 25 52 58 86 93 74 8 6 14 27 37 53 64 67 92 88 80 79 4 7 12 31 33 36 54 13 90 16 17 23 50 21 24 29 87 32 34 35 38 39 42 63 43 44 46 48 49 56 57 59 60 61 70 71 72 73 78 81 82 83 85
